primitives/keystore/src/lib.rs

```diff
@@ -17,6 +17,7 @@
 
 //! Keystore traits
 
+#[cfg(feature = "std")]
 pub mod testing;
```

Comment:
This contains `Pair` and there's not point in exposing it in no-std. 
To be fair is `Pair` is behind the `full_crypto` feature, but that is unlocked transitively and IMO there is no point exposing this detail here